OsmAnd/src/net/osmand/aidl/OsmandAidlService.java

```diff
@@ -1344,6 +1344,19 @@ public boolean getRoutePoints(List<ALatLon> route) {
 			return false;
 		}
 
+		@Override
+		public String getApplicationMode() {
```

Comment:
getApplicationMode can be misleading. Unless I'm wrong you are not getting the current application mode, but the application at the moment the route was created.
If you are interested in getting the current application mode, maybe creating a generic function getSettings that is somewhat symmetrical to the setCustomization can be an alternative.